skrf/io/general.py

```diff
@@ -249,7 +249,7 @@ def write(file, obj, overwrite = True):
         pickle.dump(obj, fid, protocol=2)
         fid.close()
 
-def read_all(dir: str ='.', contains = None, f_unit = None, obj_type=None, files: list=None, recursive=False) -> dict:
+def read_all(dir: str ='.', sort = True, contains = None, f_unit = None, obj_type=None, files: list=None, recursive=False) -> dict:
     """
```

Comment:
to be decided of the default value is True.

By the way, letting sort=True does not allow different sorting method, as you proposed initially. So, to be decided as well if further sorting method should be employed.

Any advice ?

Reply:
Thank you for your quick response.

Default: True
Of course it need to be decided, if the default value is True. No offense. Don't want to be rude.
Actually I thought a bit about it and could not find a drawback. 
For Linux users the filename import into filelist can be illogical, while for Windows users I do not expect a change.

Boolean: 
I picked up your initial idea with the boolean parameter,
I do not know other possible useful sorting methods for the time being which require an instant representation in skrf. 
I thought like this: 
A start has been made with introducing the idea of applying a sort algorithm.
If someone sees a need for a special sort implementation, they hopefully will implement or make a proposal to apply changes.

Reply:
> Default: True Of course it need to be decided, if the default value is True. No offense. Don't want to be rude. Actually I thought a bit about it and could not find a drawback. For Linux users the filename import into filelist can be illogical, while for Windows users I do not expect a change.

It is important to not brake previous code, so I'd prefer the `sort` parameter to be off by default. Thanks. 
 
> Boolean: I picked up your initial idea with the boolean parameter, I do not know other possible useful sorting methods for the time being which require an instant representation in skrf. I thought like this: A start has been made with introducing the idea of applying a sort algorithm. If someone sees a need for a special sort implementation, they hopefully will implement or make a proposal to apply changes.

Yes, eventually. So let's keep the boolean parameter, and if later someone need to sort the file by time, we will advise.